python/neuroglancer/pipeline/volumes.py

```diff
@@ -0,0 +1,151 @@
+
```

Comment:
Is this actually used anywhere?

Reply:
I saw references to the HDF5Volume in task_creation.py I was leaving it alone just in case. What do you think? I think you're more familiar with that than I am.

Reply:
I guess it's fine.... task_creation.py it's pretty outdated, we need to clean that stuff eventually.